payments/schema/types.py

```diff
@@ -269,11 +269,6 @@ class Meta:
         interfaces = (graphene.relay.Node,)
         connection_class = CountConnection
 
-    @classmethod
-    @view_permission_required(BerthSwitchOffer)
-    def get_queryset(cls, queryset, info):
-        return super().get_queryset(queryset, info)
-
```

Comment:
Now that we don't return the offer, we should add this back

Reply:
It still needs to be removed so that the customers can access (`payments/schema/mutations.py:874`):
```
        offer = get_node_from_global_id(
            info, offer_id, only_type=BerthSwitchOfferNode, nullable=False,
        )
 ```

Reply:
Does `get_node_from_global_id` check for permissions as well? I think it doesn't

Reply:
At least when testing (for example `payments.tests.test_payments_mutations_accept_berth_switch_offer.test_accept_offer`) it gives the error "You do not have permission to perform this action" when getting the instance in `utils/relay.py:20`. I think this is because it uses `BerthSwitchOfferNode.get_node` which uses `get_queryset` to fetch the model instance and that restricts the access.

Reply:
Hmmmm makes sense... let's keep a TODO that we have to enable permission checking for the node once we get the customer login sorted out.

Reply:
Sounds good. I'll add the TODO to BerthSwitchOfferNode.